package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "loopback-context",
-  "version": "3.3.0",
+  "version": "3.4.0",
   "description": "Current context for LoopBack applications, based on cls-hooked",
   "engines": {
-    "node": "^6.0 || ^8.2.1"
+    "node": "^6.0 || ^8.2.1 || ^10.14"
```

Comment:
I'm thinking whether we should just make it `>=6`. 
@strongloop/loopback-maintainers, do you know any particular reasons why we want to specify a particular node.js version? Thanks.

Reply:
i first did like that, but later changed it to keep the same way it was kept before. cls-hooked is using async_hooks api of node if node version is >=8.2.1. it's still in experimental status, so i think it's better to keep the node version under control instead of >6.0. there might be breaking changes still on this api until it becomes stable, that's my 2c.